src/components/patientPage/patientCard/index.tsx

```diff
@@ -0,0 +1,128 @@
+import React from "react";
+
+import { Callout, Icon, H3, H5 } from "@blueprintjs/core";
+import PatientInfo from "components/patientPage/patientCard/patientInfo";
+import { Patient } from "types";
+import { getPatientResources, getSubjectResources } from "services/api";
+
+import "./style.css";
+
+interface Props {
+  patient: Patient;
+}
+
+const PatientCard = ({ patient }: Props) => {
+  /*
+  getPatientNumberCard and getSubjectNumberCard are now rendering PatientInfo elements with click option which print the results on the console.
+  */
+  const getPatientNumberCard = (resourceName: string, writtenName: string) => {
+    if (patient.number !== undefined)
+      if (patient.number[resourceName] !== undefined)
+        return (
+          <div
+            onClick={async () => {
+              const response = await getPatientResources(
+                resourceName,
+                patient.id
+              );
+              console.log(writtenName + " : ", response.data.entry);
+            }}
+          >
+            <PatientInfo
+              type={writtenName}
+              content={patient.number[resourceName].toString()}
+            />
+          </div>
+        );
+  };
+
+  const getSubjectNumberCard = (resourceName: string, writtenName: string) => {
+    if (patient.number !== undefined)
+      if (patient.number[resourceName] !== undefined)
+        return (
+          <div
+            onClick={async () => {
+              const response = await getSubjectResources(
+                resourceName,
+                patient.id
+              );
+              console.log(writtenName + " : ", response.data.entry);
+            }}
+          >
+            <PatientInfo
+              type={writtenName}
+              content={patient.number[resourceName].toString()}
+            />
+          </div>
+        );
+  };
+
+  const getPatientCard = () => {
+    /*
+    Function getPatientCard
+    Get the PatientData object corresponding to patientId and generate jsx.
+    TODO: adapt this function to get data from the rest API
+
+    Return page content with patient data.
+    */
+    if (!patient) {
+      return (
+        <>
+          <Callout title="Données non chargées"></Callout>
+        </>
+      );
+    } else {
+      // case : rendering, patient found
+      return (
+        <>
+          <div className="centeredName">
+            {patient.lastName && (
```

Comment:
I'm not quite sure about these lines: what about if the patient.lastName is unknown ? what do we want to display: nothing or `Nom de famille: inconnu` ? I think in some cases we still want to display the label ("nom de famille") even if the field is missing in the patient data. what do you think ?

Reply:
Now displaying :"nom inconnu" or "inconnu" if values are unknown